react/features/participants-pane/components/web/MeetingParticipantItem.js

```diff
@@ -250,12 +260,24 @@ function MeetingParticipantItem({
  * @returns {Props}
  */
 function _mapStateToProps(state, ownProps): Object {
-    const { participantID } = ownProps;
+    const { participantID, searchString } = ownProps;
     const { ownerId } = state['features/shared-video'];
     const localParticipantId = getLocalParticipant(state).id;
 
     const participant = getParticipantByIdOrUndefined(state, participantID);
 
+    const _displayName = getParticipantDisplayName(state, participant?.id);
+
+    let _matchesSearch = false;
+    const names = _displayName.split(' ');
+
+    for (const name of names) {
+        if (name.toLowerCase().startsWith(searchString.toLowerCase())) {
```

Comment:
i'm not sure what the requirements are, but using includes rather than startsWith seems a better fit for this usecase

Reply:
The requirement is that the name should start with the search string, not include it